config/initializers/carrierwave_config.rb

```diff
@@ -2,15 +2,13 @@
 
 if Settings.s3.upload_bucket
   CarrierWave.configure do |config|
-    config.fog_provider = 'fog/aws'
-    config.fog_credentials = {
-      provider: 'AWS',
-      use_iam_profile: true
-    }
-    # This must come after fog_credentials due to
-    # https://github.com/carrierwaveuploader/carrierwave/pull/1201/files
-    config.storage = :fog
-    config.fog_directory = Settings.s3.upload_bucket
-    config.fog_public = false
+    config.fog_provider = 'fog/aws' # holdover, even though we use carrierwave-aws instead of fog now
+    # config.fog_credentials = {
+    #   provider: 'AWS',
+    #   use_iam_profile: true
+    # }
```

Comment:
Because `config.fog_provider` still **is** required, counter-intuitively, I wanted to leave the comments in there to show that the second call was not forgotten, but intentionally unused.